app/helpers/caffeine/application_helper.rb

```diff
@@ -1,4 +1,12 @@
 module Caffeine
   module ApplicationHelper
+    def favicon_path
+      path = has_asset?('favicon') ? 'favicon' : 'caffeine/admin/icons/favicon'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.